src/ripple/ledger/impl/ViewAPI.cpp

```diff
@@ -923,7 +923,7 @@ offerDelete (View& view,
     if (! sle)
         return tesSUCCESS;
     auto offerIndex = sle->getIndex ();
-    auto owner = sle->getFieldAccount160  (sfAccount);
+    auto owner = sle->getAccountID  (sfAccount);
 
     // Detect legacy directories.
     bool bOwnerNode = sle->isFieldPresent (sfOwnerNode);
```

Comment:
@miguelportilla True but it was like that in the original implementation:
https://github.com/ripple/rippled/blob/aead038215737966d45706ac088c3854614efa88/src/ripple/app/ledger/LedgerEntrySet.cpp#L1739